index.html

```diff
@@ -8,117 +8,124 @@
     <meta name="description" content="A portfolio site showing all projects I have worked on">
     <title>Arinze's Portfolio</title>
     <link rel="shortcut icon" type="image/ico" href="images/fav-icon.ico">
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/css/bootstrap.min.css">
```

Comment:
The scope of this project, for now, is to use pure/custom CSS. No styling libraries are allowed. Please remove Bootstrap